quickstart/src/joinroom.js

```diff
@@ -9,6 +9,88 @@ const $activeParticipant = $('div#active-participant > div.participant.main', $r
 const $activeVideo = $('video', $activeParticipant);
 const $participants = $('div#participants', $room);
 
+// TODO(lrivas): Review with Charlie, I am keeping this simple for now.
+class CustomMediaStream extends MediaStream {
```

Comment:
Why do we need a CustomMediaStream?

Reply:
We've already sorted out this question, but we need this for adding non-native tracks to the MediaStream object.